### ISSUE TITLE
After seven years of development [..]

### ISSUE THREAD

**Issue by mehrdaad** [avast-tl/retdec#15]
> After seven years of development, Avast open-sources its machine-code decompiler for platform-independent analysis of executable files.

I'm sorry [this](https://blog.avast.com/avast-open-sources-its-machine-code-decompiler) blog post indicates after seven years of development.  
But how on earth 11 commits?

**Comment by mewmew** [comment]
The internal git history doesn't seem to be made available with the public release.

The commit message of rev d9230cbc943783701ba9917c794910a0f3a6f730 states "Initial commit.", version 3.0, and includes 245 032 lines of code.

To peek an interest in the development that lead up to the version 3.0 release, it would be very interesting for the prior git history to be shared with the public. If that's possible or not is up to the authors to decide.

**Comment by s3rvac** [comment]
To open-source RetDec, we had to completely re-structure our original repository. Now, the code spans 26 repositories, where all of them are here on GitHub. Also, we had to replace some parts in our original code that would prevent us from open-sourcing the decompiler. Thus, we were unable to preserve our entire commit history (circa 35 000 commits).

Believe me that the initial commit did not happen overnight ;-). Also, even though I am listed as the commiter of the initial commit, the code represents work of 20-30 people. I hope this is fairly obvious, given the size of the code base.
